environment_and_metadata.rst

```diff
@@ -721,27 +721,27 @@ And the output would look like:
    OCI_CMD="bash"
    # ENTRYPOINT only - run entrypoint plus args
    if [ -z "$OCI_CMD" ] && [ -n "$OCI_ENTRYPOINT" ]; then
-   APPTAINER_OCI_RUN="${OCI_ENTRYPOINT} $@"
+   {ENVPREFIX}_OCI_RUN="${OCI_ENTRYPOINT} $@"
```

Comment:
These variables were actually changed back to `SINGULARITY_` since they are within the SIF image: https://github.com/apptainer/apptainer/commit/86d70a01fd760ac127e17adfc39842168485962c

Reply:
Ok I changed all the `{ENVPREFIX}_OCI_RUN` back to `SINGULARITY_OCI_RUN`